src/app/dim-ui/WeaponCatalystInfo.m.scss

```diff
@@ -5,23 +5,22 @@
   padding-top: 2px;
   padding-bottom: 2px;
 
-  color: $orange;
+  color: var(--theme-accent-primary);
   font-weight: bold;
 }
 
 .catalystProgress {
   display: flex;
   flex-direction: column;
-  background: #222;
+  background: var(--theme-fill-lighten);
   gap: 4px;
 
   :global(.objective-progress) {
-    background: #222;
+    background: var(--theme-fill-darken);
   }
 }
 
 .catalystProgressSection {
   padding: 4px 10px 4px;
-  background: #333;
```

Comment:
Accidental change?

Reply:
This was intentional after I applied tinted fills on the parent element. 

As mentioned in another comment, there is currently an inconsistent hierarchy of nesting/filling ItemPopup panels (weapon level, catalyst details/progress, pattern progress) as these are often combined in many permutations. 

Tinting is a more flexible solution IMO as it works better against the different yellow/purple tinted background shades for different rarity items.